src/main/java/uk/gov/hmcts/ccd/domain/types/PredefinedFieldsIDs.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.ccd.domain.types;
+
+/**
+ * IDs of fields in predefined complex types.
+ */
+public enum PredefinedFieldsIDs {
```

Comment:
better to avoid **plural names** in **class naming** conventions

Reply:
:) Let me check it